content/200-concepts/150-data-platform/03-projects/800-upgrading-plans.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: 'Upgrading plans'
+metaTitle: 'Upgrading plans'
+metaDescription: 'You can upgrade to a paid plan to increase the Prisma Data Proxy usage and the number of team members and environments that you can add to your project.'
+---
+
+<TopBlock>
+
+Each project you create in the Prisma Data Platform starts by running on a free plan (`Early Adopter` or `Free`).
+
+You can upgrade your project to a paid plan to increase the Prisma Data Proxy usage and the number of team members and environments that you can add to your project.
+
+<Admonition type="info">
+
+**Available plans**<br /><br />
+See [Plans and quotas](/concepts/data-platform/billing/plans-and-quotas) for a list and descriptions of all plans available in the Prisma Data Platform.
+
+</Admonition>
+
+Projects created before Jun 16, 2022 run on the `Early Adopter` plan and those created after that run on the `Free` plan.
```

Comment:
We could link to the Early Adopter page here but I understand it doesn't exist yet.

Reply:
I don't think it's worth it to have a completely new page for EA. I've added a section [here](https://deploy-preview-3207--prisma2-docs.netlify.app/docs/concepts/data-platform/billing/plans-and-quotas#early-adopter-plan). Is this what you have in mind?

Reply:
I've linked it to the [EA plan](https://deploy-preview-3207--prisma2-docs.netlify.app/docs/concepts/data-platform/billing/plans-and-quotas#early-adopter-plan) section that's part of the Billing > Plans and quotas page.